### PR TITLE
feat: enterIcon prop for editable Paragraph

### PR DESCRIPTION
**Pull request by kachanovskiy**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [X] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [X] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
editable Paragraph can have a custom "enter" icon (in the edit field) or no icon at all (which may be useful in case edit feels overloaded)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | iconEdit prop to configure custom "enter" icon in edit field (or no icon at all) |
| 🇨🇳 Chinese | iconEdit prop用于在编辑字段中配置自定义的 "enter"图标（或者定义这里无图标）。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/Base.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ interface EditConfig {
   onEnd?: () => void;
   maxLength?: number;
   autoSize?: boolean | AutoSizeType;
+  iconEnter?: React.ReactNode;
```
Comment (afc163) [inline components/typography/Base.tsx:44]: enterIcon

**Comment by afc163** [comment]
Conflicting

**Comment by afc163** [comment]
ping @kachanovskiy

**Comment by afc163** [comment]
图片

Test coverage is down.

**Comment by afc163** [comment]
ping @kachanovskiy

File: components/typography/index.zh-CN.md [hunk 2]
```diff
@@ -109,7 +110,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | onChange | 文本域编辑时触发 | function(event) | - |  |
 | onEnd | 按 ENTER 结束编辑状态时触发 | function | - | 4.14.0 |
 | onStart | 进入编辑中状态时触发 | function | - |  |
-| onCancel | 按 ESC 退出编辑状态时触发 | function | - |  | 
+| enterIcon | 在编辑段中自定义“enter”图标（传递“null”将删除图标） | ReactNode | &lt;EnterOutlined /> |  |
```
Comment (afc163) [inline components/typography/index.zh-CN.md:113]:
```suggestion
| enterIcon | 在编辑段中自定义“enter”图标（传递“null”将删除图标） | ReactNode | `<EnterOutlined />` | 4.17.0 |
```

**Comment by kachanovskiy** [comment]
i am experiencing problems syncing my fork to this repo and fetching upstream into my branch, so i will close this one, re-play all the changes over a fresh fork and re-submit a completely new PR. thanks for help!

**Pull request closed without merging**